linksAdapter.ts

```diff
@@ -58,9 +58,24 @@ export class LinkAdapter implements LinkSyncAdapter {
     return 1;
   }
 
-  handleHolochainSignal(signal: any): void {
-    if (this.linkCallback) {
-      this.linkCallback(signal.data.payload);
+  async handleHolochainSignal(signal: any): Promise<void> {
+    //Check if this signal came from another agent & contains a diff and reference_hash
+    if (signal.data.payload.diff) {
+      console.log("PerspectiveDiffSync.handleHolochainSignal: received a signal from another agent, checking if we can fast forward to this signal");
+      let diff = signal.data.payload.diff;
+      let diff_hash_reference = signal.data.payload.reference_hash;
+
+      await this.hcDna.call(DNA_NICK, ZOME_NAME, "fast_forward_signal", diff_hash_reference);
```

Comment:
Hm, I think we can do this call right inside the HC DNA in the signal callback there, right? That way we save us a zome call...